### PR TITLE
Upgrade gatling 3.9.5 -> 3.11.3

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Comment by vttranlina** [comment]
- [x] Run new gatling in the staging environment

**Comment by quantranhong1999** [comment]
Read it.

**Comment by vttranlina** [comment]
Quick test
IMAP: https://vttranlina.github.io/gatling_20240620/platformvalidationsimulation-20240620030521005/index.html
JMAP: https://vttranlina.github.io/gatling_20240620/pushplatformvalidationsimulation-20240620033423076/index.html

**Review by quantranhong1999 — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -31,12 +31,11 @@ lazy val root = (project in file("."))
...
-val gatlingVersion = "3.9.5"
+val gatlingVersion = "3.11.3"
 
 scalacOptions := Seq("-unchecked", "-deprecation", "-feature", "-language:postfixOps", "-Ywarn-unused:imports",
       "-Wconf:msg=Auto-application to \\`\\(\\)\\` is deprecated:s")
 
 enablePlugins(GatlingPlugin)
 
-lazy val gatlingImap = ProjectRef(uri("https://github.com/linagora/gatling-imap.git"), "gatling-imap")
+lazy val gatlingImap = ProjectRef(uri("https://github.com/vttranlina/gatling-imap.git"), "gatling-imap")
```
Comment (chibenwa) [inline build.sbt]: That way it should be marked as WIP.

Do not forget to update it back to `linagora` before merging.

**Review by Arsnael — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (Arsnael) [reply build.sbt]: I merged the changes on gatling-imap, please rebase and remove this commit, and we should be good :)

**Pull request merged**